apps/authentication-service/src/resources/clients/clientsRoutesHandlers.test.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  AUTH0_CLIENTS,
+  AUTH0_CLIENTS_BY_ID,
+  AUTH0_USER_BY_ID,
+} from "@/test/handlers";
+import { server } from "@/test/testServer";
+import { Request, Response } from "express";
+import { http, HttpResponse } from "msw";
+
+import {
+  clientsCreateV1,
+  clientsDeleteV1,
+  clientsGetV1,
+} from "@/resources/clients/clientsRoutesHandlers";
+import { exampleClient } from "@/test/testData/clients";
+import {
+  exampleUser,
+  exampleUserInfoResponse,
+  getTestToken,
+} from "@/test/testData/users";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+jest.mock("auth0", () => ({
+  ...jest.requireActual("auth0"),
+  UserInfoClient: class {
+    getUserInfo = (_token: string) => ({ data: exampleUserInfoResponse });
+  },
+}));
+
+describe("clientsRoutesHandlers", () => {
+  it("should create a new client", async () => {
+    server.use(
+      http.get(AUTH0_USER_BY_ID, () => HttpResponse.json(exampleUser)),
```

Comment:
I'd call this exampleUserWithoutClient. Every time I look at this I think "why does this need to be overridden?"